lfr/heterogeneous_single_assembly_3D/neutronics_standalone/neutronics.i

```diff
@@ -0,0 +1,1771 @@
+  []
+
+  # This assembles the 7 pins into an assembly with a duct
+  [ASM_CM]
+    type = PatternedHexMeshGenerator
+    inputs = 'PinIn_CM PinOut_CM'
+    pattern_boundary = hexagon
+    background_intervals = '1'
+    hexagon_size = ${half_asmpitch}
+    duct_sizes = '${fparse flat_to_flat/2} ${fparse flat_to_flat/2 + duct_thickness}'
+    duct_sizes_style = apothem
+    duct_intervals = '1 1'
+    background_block_id = '4'
+    duct_block_ids = '5 6'
+    external_boundary_id = 997
+    pattern = '1 1 1 1 1 1 1;
+              1 0 0 0 0 0 0 1;
+             1 0 0 0 0 0 0 0 1;
+            1 0 0 0 0 0 0 0 0 1;
+           1 0 0 0 0 0 0 0 0 0 1;
+          1 0 0 0 0 0 0 0 0 0 0 1;
+         1 0 0 0 0 0 0 0 0 0 0 0 1;
+          1 0 0 0 0 0 0 0 0 0 0 1;
+           1 0 0 0 0 0 0 0 0 0 1;
+            1 0 0 0 0 0 0 0 0 1;
+             1 0 0 0 0 0 0 0 1;
+              1 0 0 0 0 0 0 1;
+               1 1 1 1 1 1 1'
+  []
+  [coarse_mesh]
+    type = AdvancedExtruderGenerator
+    input = ASM_CM
+    heights = '${dhA} ${dhB} ${dhC} ${dhD} ${dhE} ${dhF} ${dhG} ${dhH} ${dhI} ${dhJ}'
+    num_layers = '${num_axmeshA} ${num_axmeshB} ${num_axmeshC} ${num_axmeshD} ${num_axmeshE} ${num_axmeshF} ${num_axmeshG} ${num_axmeshH} ${num_axmeshI} ${num_axmeshJ}'
+    direction = '0 0 1'
+    top_boundary = 998
+    bottom_boundary = 999
+  []
+
+  [cmesh]
+    type = CoarseMeshExtraElementIDGenerator
+    input = assign
+    coarse_mesh = coarse_mesh
+    extra_element_id_name = coarse_element_id
+  []
+  uniform_refine = 0
+[]
+
+# ==============================================================================
+# Transport Systems
+# ==============================================================================
+[TransportSystems]
+  particle = neutron
+  equation_type = eigenvalue
+  G = 9
+  VacuumBoundary = 'ASSEMBLY_TOP ASSEMBLY_BOTTOM'
+  ReflectingBoundary = 'ASSEMBLY_SIDE'
+  [sn]
+    scheme = DFEM-SN
+    family = MONOMIAL
+    order = FIRST
+    AQtype = Gauss-Chebyshev
+    NPolar = 2
+    NAzmthl = 3
+    NA = 1
+    sweep_type = asynchronous_parallel_sweeper
+    using_array_variable = true
+    collapse_scattering  = true
+  []
+[]
+
+# ==============================================================================
+# Executioner
+# ==============================================================================
+[Executioner]
+  type = SweepUpdate
+  verbose = true
+
+  richardson_rel_tol = 1e-4
+  richardson_max_its = 500
+  richardson_value = 'eigenvalue'
+  inner_solve_type = SI
+  max_inner_its = 9
+
+  cmfd_acceleration = true
+  coarse_element_id = coarse_element_id
+  cmfd_eigen_solver_type = krylovshur
+[]
+
+# ==============================================================================
+# Flux Normalization for Total Power
+# ==============================================================================
+[PowerDensity]
+  power = ${totalpower}
+  power_density_variable = power_density
+  family = MONOMIAL
+  order = CONSTANT
```

Comment:
If there is no particular reason to use constant monomial for power density, I suggest to remove `family` and `order` here. The code can automatically decide them based on flux variables.

Reply:
So, when I didn't specify them, the power variable was `Lagrange` (? if I remember correctly) which gave an artificial non-zero power in non-fuel region due to interpolation. As I used very coarse mesh (like just one radial mesh in a helium hole inside the annular fuel shape), this effect was significant. On top of that, I also observed the heat source transferred from Griffin in its sub-app had a non-physical error seemingly caused by a transfer artifact. The use of constant MONOMIAL here resolved all these issues.

Reply:
This PR is only for a Griffin standalone calculation. The transfer issue happened in the Griffin-Heat conduction multiapp system. I didn't dig into this deeply when I ran into this issue.  

In fact, I used the constant MONOMIAL type for the power variable more for the first reason. I didn't want to have non-physical heat source in non-fuel region transferred to the heat conduction module. The power distribution was obviously wrong and I don't think power need to be interpolated.

Reply:
What I was saying is that the target variable in the transfer must be ELEMENTAL as well. If we use constant monomial, we are reducing the accuracy which is typically not a big deal but concerning.

Reply:
This is the instruction from the server:
```
If this message is needed, please contact the sender and have them resend the attachment in the allowed format (e.g. filename.inl.zip)
```
@pak1sol can you rename the file and send it again?